microsoft-edge/webview2/release-notes.md

```diff
@@ -2897,6 +2900,8 @@ For full API compatibility, this version of the WebView2 SDK requires WebView2 R
 
 ###### Promotions
 
+<!-- todo: change version number in links to match heading? -->

```

Comment:
@novac42 @peiche-jessica - related to a deprecation notice precedent version number.  The version numbers in the links in this section don't match the h2 heading's version number.  We could retain the "todo" comment and decide later whether to change the version # in the API Ref links.

Reply:
keeping todo flag, not revising paths in present PR